documentation/functionalDoc/functionalDoc.tex

```diff
@@ -92,10 +92,38 @@ \subsection{Generators}
 \scriptsize{regulates the same node} & \multirow{-2}{*}{\scriptsize{true}} & \multirow{-2}{*}{\scriptsize{-}} & \scriptsize{+ VRRemote model} \\
 \bottomrule
 \end{tabular}
-\caption{Generators modelling.}
+\caption{Generators modelling when $targetV \leq TfoVoltageLevel$}
```

Comment:
Shouldn't we also update the text to say that the parameter TfoVoltageLevel has an influence on the chosen model ?